packages/libs/error-stack/src/hook.rs

```diff
@@ -292,7 +294,7 @@ impl Report<()> {
     /// </pre>
     #[cfg(feature = "std")]
     pub fn install_debug_hook_fallback(
-        hook: impl for<'a> Fn(&Frame, &mut HookContext<'a, Frame>) -> Vec<Emit> + Send + Sync + 'static,
+        hook: impl for<'a> Fn(&Frame, &mut HookContext<'a, Frame>) + Send + Sync + 'static,
```

Comment:
Untested, but the above is working
```suggestion
        hook: impl Fn(&Frame, &mut HookContext<Frame>) + Send + Sync + 'static,
```

Reply:
fixed in [7640562](https://github.com/hashintel/hash/pull/1008/commits/76405629a6351a476bb7a4be3a1591fdc26ebca2)